File: AspNetMvcUnleashed/0672329980_source/Chapter07/VB/MvcApplication1/MvcApplication1/Controllers/CustomerController.vb
Public Class CustomerController
    Inherits System.Web.Mvc.Controller

    ' GET: /Customer/Create
    Function Create() As ActionResult
        Return View()
    End Function


    ' POST: /Customer/Create
    <AcceptVerbs(HttpVerbs.Post)> _
    Function Create(ByVal customerToCreate As Customer) As ActionResult
        ' Add customer to database
        Return RedirectToAction("Index")
    End Function

End Class

File: AspNetMvcUnleashed/0672329980_source/Chapter07/CS/MvcApplication1/MvcApplication1/Controllers/CustomerController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;
using System.Web.Mvc.Ajax;
using MvcApplication1.Models;

namespace MvcApplication1.Controllers
{
    public class CustomerController : Controller
    {
        //
        // GET: /Customer/Create

        public ActionResult Create()
        {
            return View();
        } 

        //
        // POST: /Customer/Create

        [AcceptVerbs(HttpVerbs.Post)]
        public ActionResult Create(Customer customerToCreate)
        {
            // Add customer and address to database
            return RedirectToAction("Index");
        }

    }
}
